lib/components/notifications-component.js

```diff
@@ -16,8 +16,13 @@ export default class NotificationsComponent extends BaseContainer {
 		return text;
 	}
 	selectCommentByText( commentText ) {
-		const commentSelector = by.xpath( `//div[normalize-space(text())='${commentText}']` );
-		return driverHelper.clickWhenClickable( this.driver, commentSelector );
+		const self = this;
+		const commentSelector = by.css( 'div > div.wpnc__text-summary > div:nth-child(1)' );
+		this.driver.findElement( by.css( 'div.wpnc__excerpt' ) ).getText().then( function( selectedCommentText ) {
+			if( selectedCommentText === commentText ) {
```

Comment:
I also think this breaks down if the expected comment isn't first in the list.  99 times out of 100 I'm sure it will be, but if multiple copies of the test are running at the same time there could be a timing race that puts them out of order.